src/protocols/vnc/settings.c

```diff
@@ -28,13 +28,15 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#include <unistd.h>
```

Comment:
What brings this in?

Reply:
That probably was for one of the x509 bits - let me pull it out and make sure it isn't required anymore.